### PR TITLE
Update Student Apply page for Fa20 and typo fixes

### PR DESCRIPTION
**Pull request by suewee**
Update recruitment timeline with temporary interest form and social media promo. Include Fa20 FAQ questions regarding virtual recruitment. 

Fix typos in CCL and SaverLife teams. Attempt to fix GLEN World issue where project cover photo will not load on mobile by changing the image dimensions to that of KSO's.

### CODE REVIEW

File: data/studentApplyData.js [hunk 1]
```diff
@@ -195,26 +262,30 @@ const studentProcess = [
...
+  //   date: firstRoundInterviewDate
+  // }
 ];
 
 const closedText = (
   <p>
-    <b><em>Our current application period for {currentSemester} is currently open.</em></b> We are <b><em>only accepting Product Designers</em></b> for the Spring semester.
-    For those that are interested in software development, stop by our website in the future to view updated information about the next application period!
+    Our application period for {currentSemester} <b><em>will open soon. </em></b> We will be accepting 
+    <b> Academy Members, Software Developers and a Product Designer</b> for the
```
Comment (arpanlaha) [inline data/studentApplyData.js]: we shouldn't be saying anything that implies how many people we will be accepting

**Review by arpanlaha — CHANGES_REQUESTED** [review]
Looks good, just the one change in my comment

File: data/studentApplyData.js (hunk 1 above)
Comment (suewee) [reply data/studentApplyData.js]: got it! updated the wording in that instance, on the Notion page and interest form

**Review by arpanlaha — APPROVED** [review]
🍪

**Pull request merged**